doc/source/glossary.rst

```diff
@@ -257,67 +255,40 @@ Glossary
             [[1, 2, 3],
              [4, 5, 6]]
 
-       is represented in the column-major order as::
 
-           [1, 4, 2, 5, 3, 6]
+   `column-major <https://en.wikipedia.org/wiki/Row-_and_column-major_order>`_
+       \
 
-       Column-major order is also known as the Fortran order, as the Fortran
-       programming language uses it.
 
    copy
 
        See :term:`view`.
 
 
-   decorator
-       An operator that transforms a function.  For example, a ``log``
-       decorator may be defined to print debugging information upon
-       function execution::
-
-         >>> def log(f):
-         ...     def new_logging_func(*args, **kwargs):
-         ...         print("Logging call with parameters:", args, kwargs)
-         ...         return f(*args, **kwargs)
-         ...
-         ...     return new_logging_func
+   :term:`decorator <python:decorator>`
+       \
```

Comment:
Does this work / makes sense in a glossary? Where does ``:term:`decorator` `` link to?